app/config.py

```diff
@@ -438,3 +444,9 @@ def log_configuration(self):
         if self._runtime_environment.metrics_pushgateway_enabled:
             self.logger.info("Metrics Pushgateway: %s", self.prometheus_pushgateway)
             self.logger.info("Kubernetes Namespace: %s", self.kubernetes_namespace)
+
+    def build_sys_profile_fields(self) -> list:
+        if self.sp_fields_to_log == "":
+            return []
+        else:
+            return self.sp_fields_to_log.split(",")
```

Comment:
Splitting an empty string returns an empty array by default
```suggestion
        return self.sp_fields_to_log.split(",")
```

Reply:
At the end I removed this methods as it was not necessary